website/docs/releases.mdx

```diff
@@ -9,6 +9,39 @@ import TierLabel from "./_components/TierLabel";
 :::note
 This page details the changes for Weave GitOps Enterprise and its associated components. For Weave GitOps OSS - please see the release notes on [GitHub](https://github.com/weaveworks/weave-gitops/releases).
 :::
+## v0.16.0
+2023-02-02
+### Highlights
+
+#### Create External Secrets via WGE UI
+- It's becoming easier to create new a external secret CR through the UI instead of writing the whole CR yaml.
+- The creation form will help users choose which cluster to deploy the External Secret to and which secret store to sync the secrets from. 
+- It's all done in the GitOps way. 
+
+#### Show PR Links from Pipline details
+- Users will be able to follow a link in the Pipeline Details Page to view open pull requests from given environment within their pipeline.
+
```

Comment:
```suggestion
```

Reply:
Unfortunately this feature is not quite ready, we've done the back end work for [this](https://github.com/weaveworks/weave-gitops-enterprise/issues/2216) but not the [UI ](https://github.com/weaveworks/weave-gitops-enterprise/issues/2184)yet